Apps/Sandcastle/gallery/Terrain.html

```diff
@@ -34,16 +34,19 @@
 //Sandcastle_Begin
 var viewer = new Cesium.Viewer('cesiumContainer');
 
-var scene = viewer.scene;
-var globe = scene.globe;
-globe.depthTestAgainstTerrain = true;
-
 var cesiumTerrainProviderMeshes = new Cesium.CesiumTerrainProvider({
     url : '//assets.agi.com/stk-terrain/world',
     requestWaterMask : true,
     requestVertexNormals : true
 });
+viewer.terrainProvider = cesiumTerrainProviderMeshes;
+
+var mountEverest = new Cesium.Cartesian3(300770.50872389384, 5634912.131394585, 2978152.2865545116);
```

Comment:
Duplicating the initial view code like this is ugly.  The reason it was broken with previous changes is because Sandcastle currently expects the first button of the first menu to be the default option.  If you want to give me access to your fork, I can tweak the Sandcastle code to avoid having this here.  Otherwise I can just do it after we merge this into master.